spec/helpers/error_items_helper_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+RSpec.describe ErrorItemsHelper do
+  describe "#error_items" do
+    let(:empty_error_messages) { {} }
+    let(:error_messages) { { title: ["can't be blank"], path: ["must be a valid path starting with a /"] } }
+
+    describe "when valid" do
+      it "does not format errors" do
+        expect(error_items(empty_error_messages, :title)).to eq(nil)
+      end
+    end
+
+    describe "when invalid" do
+      it "formats the error message for a blank title field" do
+        expect(error_items(error_messages, :title)).to eq("Title can't be blank")
+      end
+
+      it "formats the error message for an invalid path field" do
```

Comment:
Love to have: I think it would be great if this test tested the output when a field has two errors.
e.g. if 

```
path: ["must be a valid path starting with a /", "must do something else"]
```

Reply:
done!